src/customer/strategies/googlepay/googlepay-customer-strategy.ts

```diff
@@ -139,7 +154,7 @@ export default class GooglePayCustomerStrategy implements CustomerStrategy {
     }
 
     private _onPaymentSelectComplete(): void {
-        this._formPoster.postForm('/checkout.php', {
+        this._formPoster.postForm(`${this._StoreUrl}`, {
```

Comment:
Think we don't need backtick quote here.

Reply:
sure @bc-peng, I made changes, It's ready for another review ♻️